frontend/public/module/k8s/types.ts

```diff
@@ -1113,12 +1113,33 @@ export type NetworkPolicyPort = {
 export type ConsolePluginKind = K8sResourceCommon & {
```

Comment:
I believe a number of these fields should be optional, but don't have a `?` in the type definition.